.github/ISSUE_TEMPLATE/bug.yml

```
@@ -0,0 +1,62 @@
+name: Bug report
+description: Submit a bug report
+labels: ["type-bug"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        # Bug report
+
+        ## New to Python?
+
+        For help or advice on using Python, try one of the following options instead of filing a GitHub issue:
+
+          - Posting on [Discourse](https://discuss.python.org/c/users/7)
+          - Reading the [Python tutorial](https://docs.python.org/3/tutorial/)
+          - Emailing [python-list](https://mail.python.org/mailman/listinfo/python-list)
+  - type: checkboxes
+    attributes:
+      label: Checklist
+      description: A bug in a third-party project (e.g. `pip` or `requests`) should be reported to that project's issue tracker, not CPython
```

I suggest avoiding Latin abbreviations, they can be confusing for people with English as a second language, and don't always play nice with screen readers. Plain English is usually more accessible for everyone.

```suggestion
      description: A bug in a third-party project (for example, `pip` or `requests`) should be reported to that project's issue tracker, not CPython
```

Or "such as".

Here's a couple of tech style guides:

* https://developers.google.com/style/abbreviations#dont-use
* https://learn.microsoft.com/en-us/style-guide/word-choice/use-us-spelling-avoid-non-english-words

And a couple from governments, who want to be understood by all:

* https://www.gov.uk/guidance/style-guide/a-to-z-of-gov-uk-style#eg-etc-and-ie
  * Background: https://insidegovuk.blog.gov.uk/2016/07/20/changes-to-the-style-guide-no-more-eg-and-ie-etc/
* https://www.stylemanual.gov.au/grammar-punctuation-and-conventions/shortened-words-and-phrases/latin-shortened-forms